src/library/scala/collection/StringOps.scala

```diff
@@ -1523,7 +1523,7 @@ final class StringOps(private val s: String) extends AnyVal {
   def permutations: Iterator[String] = new WrappedString(s).permutations.map(_.unwrap)
 }
 
-case class StringView(s: String) extends AbstractIndexedSeqView[Char] {
+final case class StringView(s: String) extends AbstractIndexedSeqView[Char] {
```

Comment:
Would this technically break source compat? If so we should target 2.13.0.

Reply:
I believe this should target 2.13.0 anyway (based on the linked issue) - I just forgot to update the milestone